docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -363,53 +390,79 @@ Reboot the flight controller and connect to it again. This time in the "Messages
 
     Message Showing Outputs Are Using DSHOT
 
-Testing the Motor with Mission Planner
-**************************************
-.. warning:: Double check that the motor is secured and there is no propeller attached before performing any testing.
+Configuring the Safety Switch
+*******************************
+By default, ArduCopter will not allow you to test spinning your modules unless a safety switch attached to your flight controller is set. If you have a safety switch connected to your flight
+controller then you can move on to testing the module. **If you do not have a safety switch but would stil like to test your modules, this section will cover how to override the requirement 
+for a safety switch**.
+
+The `ArduCopter documentation <https://ardupilot.org/copter/docs/common-safety-switch-pixhawk.html>`_ provides details on how to configure the safety switch. To disable the switch,
+Set **BRD_SAFETYENABLE (also called BRD_SAFETY_DEFLT on newer firmwares) to 0**. The image below shows BRD_SAFETYENABLE set to this value in Mission Planner. If your module is
+powered on and attached to the flight controller when you change this parameter, you may hear the module arm. That is because the flight controller will begin sending 0% throttle commands
+on its outputs once this is changed.
+
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/mp_brd_safety.png
+    :align: center
+
+    Disabling the Safety Switch with the BRD_SAFETYENABLE Parameter
+
+Testing the Module with Mission Planner
+*****************************************
+.. warning:: Double check that the module is secured and there is no propeller attached before performing any testing.
 
-Now we can use Mission Planner's built-in motor testing tools to make sure the flight controller can control the motor.
+Now we can use Mission Planner's built-in motor testing tools to make sure the flight controller can control the module.
 
-1. Start with the motor powered off, but connected to the flight controller.
+1. Start with the module powered off, but connected to the flight controller.
 2. Select "Setup" from the top toolbar, and then expand "Optional Hardware" on the sidebar
-3. Select "Motor Test" from the "Optional Hardware" options.
-4. Power on the motor and wait for it to complete its startup song.
+3. Select "Motor Test" from the "Optional Hardware" options, see the image below as an example of what the Motor Test screen should look like.
```

Comment:
make 2 sentences. "...options. See the image below...should look like at this point."